misc/null-resource-with-tf-version/main.tf

```diff
@@ -0,0 +1,5 @@
+terraform {
```

Comment:
Consider a more specific name for the module
`null-resource-with-version-constraints` or just `module-with-version-constraints`